_site/page4/index.html

```diff
@@ -285,7 +310,7 @@ <h2 class="post-title">
       <a href="/blog/tags/backbone"
          title="Articles about Backbone">Backbone</a>,
       <a href="/blog/tags/security"
-         title="Articles about Security">Security</a><time class="post-date" datetime="2015-12-15">
+         title="Articles about Security">Security</a><time class="post-date" datetime="2015-12-16">
```

Comment:
@etagwerker there's a number of unrelated modifications related to timezones, you probably regenerated the blog using Ruby 2.4 or 2.5. 

I have to regenerate it using Ruby 2.3, otherwise it doesn't respect the timezone set in `config.yml` for some reason. This is fixed by #144.

Reply:
@mauro-oto ah, weird! I just regenerated it with Ruby 2.3.1